sessionctx/variable/session.go

```diff
@@ -406,6 +410,15 @@ func (s *SessionVars) GetStatusFlag(flag uint16) bool {
 	return s.Status&flag > 0
 }
 
+// GetHashJoinConcurrency gets the value of SessionVars.HashJoinConcurrency.
+func (s *SessionVars) GetHashJoinConcurrency() uint {
```

Comment:
can we remove this func and initialize `HashJoinConcurrency` in other places, thus we can use `HashJoinConcurrency` like other session variables ?